nixos/modules/programs/atop.nix

```diff
@@ -123,11 +123,11 @@ in
       boot.extraModulePackages = [ (lib.mkIf cfg.netatop.enable cfg.netatop.package) ];
       systemd =
         let
-          mkSystemd = type: cond: name: restartTriggers: {
-            ${name} = lib.mkIf cond {
+          mkSystemd = type: cond: name: restartTriggers: otherSystemdOptions: {
```

Comment:
instead of this extra argument...

Reply:
Thanks for the review. I agree that it would be cleaner to avoid this extra otherSystemdOptions parameter and I have attached an updated commit for your review.